src/rendering/renderers/gl/shader/program/ensureAttributes.ts

```diff
@@ -27,7 +27,6 @@ export function ensureAttributes(
 
         if (attributeData)
         {
-            attribute.location ??= attributeData.location;
             attribute.format ??= attributeData.format;
             attribute.offset ??= attributeData.offset;
             attribute.instance ??= attributeData.instance;
```

Comment:
BTW, do we really need this? If the geometry is used for the first time with a shader that only uses a subset of the attributes this won't work. And if the geometry already specified format offset and instance correctly, we get a warning anyway, even though no filling-in is needed.